SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift

```diff
@@ -473,7 +473,7 @@ class TransmissionViewController: NSViewController, NSTableViewDataSource, NSTab
     let propertyIndex = getCellRow(forTextField: sender)
     let target = TransmissionViewController.targetsShared!.transmissionTargets[selectedTarget!]!
     propertyValues[selectedTarget!]![propertyIndex] = sender.stringValue
-    let value = sender.stringValue.isEmpty ? nil as String! : sender.stringValue
+    let value = sender.stringValue.isEmpty ? nil as String? : sender.stringValue
```

Comment:
There is no need for this chain, stringValue cannot be nil.
`let value = sender.stringValue`
if you are worried about nil in sender, you can use: `sender?.stringValue`